test/verify/check-packagekit

```diff
@@ -218,14 +233,32 @@ class TestUpdates(MachineCase):
         b.wait_in_text("#state", "Applying updates")
         b.wait_present("#app div.progress-bar")
 
-        # should have succeeded
-        b.wait_in_text("#state", "No updates pending")
-        # empty state visible in main area
-        b.wait_present(".container-fluid div.blank-slate-pf")
+        # should have succeeded and show restart
+        b.wait_present("#app .container-fluid h1")
+        b.wait_in_text("#app .container-fluid h1", "Restart Recommended")
+        b.wait_present("#app .container-fluid button.btn-default")
 
         # new versions are now installed
         m.execute("test -f /stamp-norefs-bin-2-1 && test -f /stamp-norefs-doc-2-1")
 
+        # do the reboot; this will disconnect the web UI
+        m.reset_reboot_flag()
+        b.click("#app .container-fluid button.btn-primary")
+        b.switch_to_top()
+        b.wait_visible(".curtains-ct")
```

Comment:
I meant that we should wait for a `.curtains-ct` to be present first. Is there one always in the html? If there is none present, `wait_visible` will fail immediately, since [it expects](https://github.com/cockpit-project/cockpit/blob/3389c804e0484f5a1edc49bf2355dbad4caff4af/test/common/phantom-lib.js#L142) an element to be present.